bot/exts/filtering/_filter_lists/antispam.py

```diff
@@ -98,7 +98,7 @@ async def actions_for(
         if not current_infraction or new_infraction.infraction_type.value < current_infraction.value:
             # Pick the first triggered filter for the reason, there's no good way to decide between them.
             new_infraction.infraction_reason = (
-                f"{triggers[0].name.replace('_', ' ')} spam – {ctx.filter_info[triggers[0]]}"
+                f"{triggers[0].name.replace('_', ' ')} spam - {ctx.filter_info[triggers[0]]}"
```

Comment:
Not necessary

Reply:
Do you think it would be worth a noqa just for this? The rule is to standardise symbols to ascii, so replacing the en dash with a hyphen

Reply:
We can do a noqa, but so far all the rule has done is make us as noqas

Reply:
I'd prefer to keep it that way. Non-standard character sets aren't always obvious when reviewing PRs, so they could unknowingly get into the code base and cause confusion/issues. I'd rather have to noqa as needed, and in instances such as this just follow the rule.